integration_test/start.test.ts

```diff
@@ -0,0 +1,193 @@
+import { ChildProcess, execSync, spawn } from 'child_process';
+import * as path from 'path';
+
+import BigNumber from 'bignumber.js';
+import { TransactionObject } from '@openst/mosaic-contracts/dist/interacts/types';
+import { FacilitatorConfig } from '../src/Config/Config';
+
+import Utils from './Utils';
+import MosaicConfig from '../src/Config/MosaicConfig';
+import { default as SrcUtils } from '../src/Utils';
+
+const Web3 = require('web3');
+
+const facilitatorInit = path.join(__dirname, 'facilitator_init.sh');
+const facilitatorStart = path.join(__dirname, 'facilitator_start.sh');
+
+const originWeb3 = new Web3('http://127.0.0.1:41515');
+const auxiliaryWeb3 = new Web3('http://127.0.0.1:41000');
+const exportWorkerPrefix = 'MOSAIC_ADDRESS_PASSW_';
+
+describe('facilitator start', () => {
+  const mosaicConfig = MosaicConfig.fromFile(path.join(__dirname, 'mosaic.json'));
+
+  const ostComposer: string | undefined = mosaicConfig.originChain.contractAddresses.ostComposerAddress;
+
+  const auxChainId = 1000;
+  const outputOptions = [process.stdout, process.stderr];
+
+  let facilitatorConfig: FacilitatorConfig;
+  let originWorker: string;
+  let utils: Utils;
+  let simpleTokenInstance: any;
+  let stakerAccount: any;
+  let originFunder: string;
+  let auxiliaryFunder: string;
+  let auxWorkerExport;
+  let facilitatorStartChildProcess: ChildProcess;
+  let originAnchorInterval: any;
+  const stakerERC20Balance = 20000;
```

Comment:
what is this variable for?

Reply:
It is the staker erc20 balance. Changed the name to stakerOSTBalance for readability.

Reply:
Balance can be taken from geth right? 
Do you mean staker ost funding amount?